packages/docs/docs/careplans/tasks/tasks.md

```diff
@@ -0,0 +1,140 @@
+---
+tags:
+  - care plans
+  - workflow
+  - tasks
+---
+
+# Using `Tasks` to Manage Workflow
+
+## Introduction
+
+Workflow is an essential part of healthcare, and healthcare operations requiring coordination of many manual steps physicians, patients, nurses, care coordinators, etc.
+
+While the majority of FHIR resources represent clinical data that is _operated on_, FHIR also defines a set of workflow resources that describe and track _work to be done._ This guide will discuss the usage of the [`Task`](/docs/api/fhir/resources/task) resource, which is the basic building block resource for tracking workflow progress.
+
+A common application is for organizations to build **task queue systems** to route tasks to the correct practitioner based on specialty, level of credential, and availability. The [Medplum Task Demo](https://github.com/medplum/medplum-task-demo) application provides a minimalist task queue that demonstrates task search, assignment, and status.
+
+## Task type
+
+The `Task.code` element is used to represent the task type, equivalent to the task title. This can either be different from task to task, or selected from an standard set of task types. Using the latter approach helps enable querying across all `Tasks` of the same type.
```

Comment:
Example of Task.code will also be helpful - perhaps in accordance with SNOMED or LOINC.

Reply:
Example added with disclaimer - most of the FHIR examples just use the `code.text` field, rather than a specific SNOMED/LOINC code. I don't want users trying to bend over backwards trying to find the right code, as the task types are often implementation specific